### PR TITLE
Add cypress testing (just install and config)

### PR DESCRIPTION
**Pull request by eamyzzaj**
I installed cypress to our dependencies, added a script to run it (**npm run cy:open)**, and configured it. To prevent build errors that occurred previously , package-lock was removed and added to .gitignore. For actual apps, this is not good practice. Received OK from professor to do so for this project/class.

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
```
Comment (TLMatthies) [inline cypress.config.js:1]: One issue that might arise is that we use imports instead of require.

File: cypress/fixtures/example.json
Comment (TLMatthies) [inline cypress/fixtures/example.json:1]: What is this file for?

File: cypress/support/commands.js
Comment (TLMatthies) [inline cypress/support/commands.js:1]: Is this just an example file?

File: package-lock.json
[the remark this answers is not in the document]
Comment (J-Dees) [reply package-lock.json:1]: Yay to removing merge conflicts!

File: cypress/fixtures/example.json
Comment (eamyzzaj) [reply cypress/fixtures/example.json:1]: This was also auto-populated. If it becomes unnecessary when later running tests, I will remove. Thanks

File: cypress/support/commands.js
Comment (J-Dees) [inline cypress/support/commands.js:1]: You can use 
/*

*/ 
for long comments

File: .gitignore
[the remark this answers is not in the document]
Comment (eamyzzaj) [reply .gitignore:1]: Nice!

File: cypress/support/commands.js
Comment (eamyzzaj) [reply cypress/support/commands.js:1]: See prev. comments

**Review by TLMatthies — APPROVED** [review]

File: package-lock.json
[the remark this answers is not in the document]
Comment (eamyzzaj) [reply package-lock.json:1]: Sowwy :(

**Comment by J-Dees** [comment]
# Pull Request #65 Review:
Looks good to commit. I was reading the commits backwards, so I confirmed with Jazzy that the package-lock.json was getting removed by the final commit.

Once we have the authentication in place, we should do some Cypress testing

**Review by J-Dees — APPROVED** [review]

**Review by Mcray3000 — APPROVED** [review]

File: cypress/support/commands.js
Comment (eamyzzaj) [reply cypress/support/commands.js:1]: Will do

**Comment by Alaxxn** [comment]
This looks great! I appreciate the clear explanation for the package-lock decisions. If the professor says it's okay, then I'm sure we can handle the trade-off of managing our dependencies to ensure we keep the project moving smoothly. Seems like you've already had some feedback, and everything looks good to me. Nice job!

**Pull request merged**